content/pages/submit.rst

```diff
@@ -66,14 +66,28 @@ motorcycles, scooters), narrow track and tilting vehicles, unicycles, dicycles
 human control, rider properties, handling qualities, tires, experiments,
 aerodynamics, simulators, nonholonomic dynamics, robot riders, path following.
 
-The Evolving Scholar/Orvium
-===========================
+How To Submit Via Orvium
+========================
 
 Conference attendees will create an account on our Evolving Scholar Orvium_
 conference community page. There you will be able to submit abstracts and
 papers as well as see anonymized reviews. We will share the page and login
 instructions at a later date.
 
+Steps to submit:
+
+1. Download the template below and draft your abstract or paper.
+2. Create an account on Orivum_.
```

Comment:
Add a step that says click "new publication" to submit an abstract or paper. Andrew pointed out that it wouldn't be obvious to press "new publication" if you are submitting an abstract.